### ISSUE TITLE
Typo, Command chapter.

### ISSUE THREAD

**Issue by Layarion** [munificent/game-programming-patterns#366]
Original text: we need to turn those direct calls to jump() and fireGun() into something that we can swap out.

"direct calls to jump" needs to be changed to "direct calls of" or perhaps "direct calls for"

**Comment by JensB-tech** [comment]
This is incorrect. "Direct calls to" is perfectly valid programming phrasing. He is talking about particular method calls. When we talk about method and function calls, we say that a call is a call **to** a particular function. So no, it is not a typo and it doesn't need to be changed.

**Comment by Layarion** [comment]
> This is incorrect. "Direct calls to" is perfectly valid programming phrasing. He is talking about particular method calls. When we talk about method and function calls, we say that a call is a call **to** a particular function. So no, it is not a typo and it doesn't need to be changed.

hmm it makes sense after you explained it, but you need to consider the person reading this might not have that specific "programmer" lingo baked into them. This targets people who already don't know enough that they need patterns spelled out to them, people like me, and I even thinking it's a typo kinda proves that point already (and i am a programmer, so i mean - who are you really saying that "to" knowledge is for?). Is your target audience only for programmers who know how a programmer thinks? or for a wider audience?
